src/gui/static/src/app/components/layout/hardware-wallet/hw-passphrase-activation-dialog/hw-passphrase-activation-dialog.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA, MatDialog, MatDialogConfig } from '@angular/material/dialog';
+import { HwWalletService, OperationResults } from '../../../../services/hw-wallet.service';
+import { ChildHwDialogParams } from '../hw-options-dialog/hw-options-dialog.component';
+import { HwDialogBaseComponent } from '../hw-dialog-base.component';
+import { HwPassphraseHelpDialogComponent } from '../hw-passphrase-help-dialog/hw-passphrase-help-dialog.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163608)
```suggestion
import { HwPassphraseHelpDialogComponent } from "../hw-passphrase-help-dialog/hw-passphrase-help-dialog.component";
```